components/engine_traits/src/lib.rs

```diff
@@ -254,8 +254,6 @@
 #![feature(min_specialization)]
 #![recursion_limit = "200"]
```

Comment:
This was set for `quick_error`, is it still necessary?

Reply:
no